library/src/main/java/com/novoda/downloadmanager/lib/Cursors.java

```diff
@@ -0,0 +1,27 @@
+package com.novoda.downloadmanager.lib;
+
+import android.database.Cursor;
+
+final class Cursors {
+
+    private Cursors() {
+        // no instances
+    }
+
+    public static int getInt(Cursor c, String column) {
+        return c.getInt(columnIndexFor(c, column));
+    }
+
+    public static long getLong(Cursor c, String column) {
+        return c.getLong(columnIndexFor(c, column));
+    }
+
+    public static String getString(Cursor c, String column) {
```

Comment:
The main idea is to simplify the use of `Cursor` which is awkward at least.

You don't need to mock this method, test wise you've the same problem (using cursor will make that part of the code hard to test), it just removes noise from the client.